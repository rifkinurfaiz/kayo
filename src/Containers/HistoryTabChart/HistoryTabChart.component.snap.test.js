import * as React from 'react';
import { shallow } from 'enzyme';

import HistoryTabChartComponent from './HistoryTabChart.component';

describe('#HistoryTabChartComponent', () => {
  it('should render correctly', () => {
    expect(shallow(<HistoryTabChartComponent />)).toMatchSnapshot();
  });
});
