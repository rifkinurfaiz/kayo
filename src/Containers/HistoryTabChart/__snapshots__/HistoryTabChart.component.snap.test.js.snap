// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`#HistoryTabChartComponent should render correctly 1`] = `
<View>
  <Text>
    History tab chart
  </Text>
</View>
`;
